docs/source/operations/howto/connect-organizations.rst

```diff
@@ -0,0 +1,98 @@
+
+In this setup we want MyOrg1 to exchange assets with MyOrg2.
+We will assume that you have two values files with the configuration for your existing deployment, one for each organization named: ``backend-1-values.yaml``, ``backend-2-values.yaml``.
+
+Configure matching values for your 2 :term:`Organizations <Organization>`:
+
+#. Create an account for MyOrg2 on MyOrg1.
+   In the ``backend-1-values.yaml`` file, add the following content:
+
+   .. code-block:: yaml
+
+      addAccountOperator:
+        incomingOrganizations:
+          - name: ORG2NAME
+            secret: SECRET_ORG1_ORG2
+            channel: CHANNEL
+
+   | Replace ``ORG2NAME`` by the name of the organization as defined with the value ``organizationName`` in the second organization values file. In our example it would be ``MyOrg2``.
+   | Replace ``SECRET_ORG1_ORG2`` with the password that MyOrg2 will need to use to download assets from MyOrg1.
+   | Replace ``CHANNEL`` with the name of a :term:`Channel` MyOrg1 and MyOrg2 are part of. This needs to be one of the channels defined in the :ref:`Backend channel configuration <backend-channel-config>`. 
+
+#. Create an account for MyOrg1 on MyOrg2.
+   In the ``backend-2-values.yaml`` file add the following content:
+
+   .. code-block:: yaml
+
+      addAccountOperator:
+        incomingOrganizations:
+          - name: ORG1NAME
+            secret: SECRET_ORG2_ORG1
+            channel: CHANNEL
+
+   | Replace ``ORG1NAME`` by the name of the organization as defined with the value ``organizationName`` in the first organization values file. In our example it would be ``MyOrg1``.
+   | Replace ``SECRET_ORG2_ORG1`` with the password that MyOrg1 will need to use to download assets from MyOrg2.
+   | Replace ``CHANNEL`` with the name of a :term:`Channel` MyOrg1 and MyOrg2 are part of. This needs to be one of the channels defined in the :ref:`Backend channel configuration <backend-channel-config>`. 
+
+#. Configure MyOrg1 to use the right password when connecting to MyOrg2.
+   In the ``backend-1-values.yaml`` file add the following content under the ``addAccountOperator`` key:
+
+   .. code-block:: yaml
+ 
+      outgoingOrganizations:
+        - name: ORG2NAME
+          secret: SECRET_ORG2_ORG1
+
+   | Replace ``ORG2NAME`` with the name of the organization. In our example it would be ``MyOrg2``.
+   | Replace ``SECRET_ORG2_ORG1`` with the password defined for MyOrg1 in ``backend-2-values.yaml``.
+
+#. Configure MyOrg2 to use the right password when connecting to MyOrg1.
+   In the ``backend-2-values.yaml`` file add the following content under the ``addAccountOperator`` key:
+
+   .. code-block:: yaml
+  
+      outgoingOrganizations:
+        - name: ORG1NAME
+          secret: SECRET_ORG1_ORG2
+
+   | Replace ``ORG1NAME`` with the name of the organization. In our example it would be ``MyOrg1``.
+   | Replace ``SECRET_ORG1_ORG2`` with the password defined for MyOrg2 in ``backend-1-values.yaml``.
```

Comment:
Since all this conf is a bit confusing, I think it would be very useful to add a summary. Something like:

Your configuration files should look like this:

```yaml
      # backend-1-values.yaml
      addAccountOperator:
        incomingOrganizations:
          - name: ORG2NAME
            secret: SECRET_ORG1_ORG2
            channel: CHANNEL
        outgoingOrganizations:
          - name: ORG2NAME
            secret: SECRET_ORG2_ORG1
```

```yaml
      # backend-2-values.yaml
      addAccountOperator:
        incomingOrganizations:
          - name: ORG1NAME
            secret: SECRET_ORG2_ORG1
            channel: CHANNEL
        outgoingOrganizations:
          - name: ORG1NAME
            secret: SECRET_ORG1_ORG2
```